precommit.sh

```diff
@@ -1,5 +1,6 @@
+#!/bin/sh
 # Auto format.
-docker exec tcf_django autopep8 --in-place --jobs=0 --aggressive --aggressive -r tcf_website tcf_core
+docker exec tcf_django autopep8 --in-place --jobs=0 --max-line-length=100 --aggressive --aggressive -r tcf_website tcf_core
```

Comment:
How did you decide exactly 100 characters was the right amount? (side note, 79 is such a weird default limit)

Reply:
I believe 80, 100, and 120 are some popular choices--[this is the only source I could find, though](https://www.quora.com/Why-does-Google-Java-coding-style-guide-chose-maximum-of-100-characters-column-limit-per-line-why-not-120-just-curious-to-know)--but based on our reaction to `autopep8`'s autoformatting, I think there is a consensus that 80 is too short. Also, 120 is not too long for widescreen monitors, but given that most of us are students developing on a laptop, I chose 100. Does this make sense?

As a side note, [79 is mentioned in Python PEP.](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
Sounds good to me!